website/src/_posts/2022-03-2.4-2.7.md

```diff
@@ -10,15 +10,15 @@ image: "https://uppy.io/images/blog/2.4-2.7/compressor.jpg"
 published: true
 ---
 
-Recent Uppy updates brings image compression, improved Transloadit rate limiting and lighter Dashboard plugin. We’ve also moved our end to end tests to Cypress, and are slowly converting plugins to ESM modules.
+We're always looking for opportunities to teach Uppy cool new tricks, and the past few months have been no exception. Since our [most recent post](https://uppy.io/blog/2021/12/2.1-2.3/) in December, Uppy has continued to receive a steady stream of updates. This post covers all the improvements made in Uppy versions 2.4 through 2.7. Changes and additions include: image compression, improved Transloadit rate limiting and a lighter Dashboard plugin. We’ve also moved our end-to-end tests to Cypress, and are slowly converting plugins to ESM modules.
```

Comment:
Something that bothered me since this post landed: ESM means `ECMAScript Modules`, so "ESM modules" would be…
```suggestion
We're always looking for opportunities to teach Uppy cool new tricks, and the past few months have been no exception. Since our [most recent post](https://uppy.io/blog/2021/12/2.1-2.3/) in December, Uppy has continued to receive a steady stream of updates. This post covers all the improvements made in Uppy versions 2.4 through 2.7. Changes and additions include: image compression, improved Transloadit rate limiting and a lighter Dashboard plugin. We’ve also moved our end-to-end tests to Cypress, and are slowly converting plugins to ES modules.
```

Reply:
> Something that bothered me since this post landed: ESM means `ECMAScript Modules`, so "ESM modules" would be…

While you are undoubtedly correct... are you confident readers will be familiar with this term and understand what is meant? Sometimes these incorrect abbreviations become common parlance, so when you correct them, people no longer know what it means.

I'm not familiar with the term myself at all, so I can't really judge 😄

Reply:
I'm confident that 100% of readers who understand what we meant by `ESM modules` would also understand `ES modules (ESM)` :)

Reply:
Haha, alright. Just checking. Sounds like a good change then!